test/logic/solana_logic_test.rb

```diff
@@ -269,17 +269,28 @@ def validators_info
     end
   end
 
-  test 'validators_info_save' do
+  test '#validators_info_save updates persisted validators' do
     #  To skip validators_info_get
     payload = @mainnet_initial_payload.merge(
       validators_info: @validators_info
     )
 
+    create(
+      :validator,
+      :mainnet,
+      name: 'test_name',
+      keybase_id: 'test_keybase_id',
+      details: 'test_details',
+      avatar_url: 'test_avatar_url',
+      www_url: 'test_www_url',
```

Comment:
To simulate real life example you should create a validator without name, keybase_id, details, avatar etc. These data should be completed after script finishes.